independent-projects/revapi/src/main/resources/revapi/revapi-configuration.xml

```diff
@@ -3,7 +3,7 @@
 <analysisConfiguration>
   <revapi.java>
     <missing-classes>
-      <behavior>error</behavior>
```

Comment:
why this change ?

Reply:
So that it should report missing classes not fail the build